specification/schemas/circuit.schema.json

```diff
@@ -1,9 +1,12 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
   "$id": "/tqec/specification/schemas/circuit.schema.json",
-  "$comment": "For the moment, this field is not fixed. Anything compatible with cirq is a good input. If this is something compatible with criq, we will also need a way to get information on detectors, observables and shift_coords that are not part of cirq official gates.",
   "title": "Quantum circuit",
   "description": "A quantum circuit representing a quantum computation.",
   "type": "object",
-  "properties": {}
-}
+  "properties": {
+    "openqasm": {
```

Comment:
Is this the qasm representation of the circuit?

https://quantumai.google/cirq/build/interop#importing_from_openqasm

Reply:
This is, for the moment, the OpenQASM representation of the circuit. As noted in https://github.com/QCHackers/tqec/issues/38#issuecomment-1885885099:
> OpenQASM (2.0 I guess?) is fine on my side. I have no strong feeling about that part, anything that fits the needs (easy to generate on the frontend, easy to parse on the backend and able to represent everything that we need) is fine for me.

So anything in the following list should be fine:
- OpenQASM 2.0 (I just saw that the `cirq` importer seems to not support `reset`, this might be an issue)
- Quirk visual format
- Any `stim`-compatible format (textual, json, etc.)

Anything **not** in the previous list is up to debate.